docs/canonical-ways-to-investigate-bugs.md

```diff
@@ -0,0 +1,368 @@
+{"message":"libcore-call: DynamicObject#putString","data":["BLOCKCHAIN_EXPLORER_VERSION","v2"],"level":"debug","timestamp":"2020-03-03T15:12:36.046Z"}
+{"message":"libcore-result: DynamicObject#putString","data":{"value":{}},"level":"debug","timestamp":"2020-03-03T15:12:36.046Z"}
+{"message":"libcore: getOrCreateWallet xpub6DBXGC1faXnWSWofjAXjfxihNmYEcJj7KcWytuc7AZK39zS8KdgbosCjnPyHRbwKxcJxnWbzLwZYaPHFE3zoQZmGDbPzwRuoSfFaeSTVwUT_zencash_","level":"debug","timestamp":"2020-03-03T15:12:36.046Z"}
+{"message":"libcore-call: WalletPool#getWallet","data":["xpub6DBXGC1faXnWSWofjAXjfxihNmYEcJj7KcWytuc7AZK39zS8KdgbosCjnPyHRbwKxcJxnWbzLwZYaPHFE3zoQZmGDbPzwRuoSfFaeSTVwUT_zencash_"],"level":"debug","timestamp":"2020-03-03T15:12:36.046Z"}
+{"message":"libcore-result: WalletPool#getWallet","data":{"value":{}},"level":"debug","timestamp":"2020-03-03T15:12:36.052Z"}
+{"message":"libcore-call: WalletPool#updateWalletConfig","data":["xpub6DBXGC1faXnWSWofjAXjfxihNmYEcJj7KcWytuc7AZK39zS8KdgbosCjnPyHRbwKxcJxnWbzLwZYaPHFE3zoQZmGDbPzwRuoSfFaeSTVwUT_zencash_",{}],"level":"debug","timestamp":"2020-03-03T15:12:36.052Z"}
+{"message":"libcore-result: WalletPool#updateWalletConfig","data":{"value":32},"level":"debug","timestamp":"2020-03-03T15:12:36.063Z"}
+{"message":"libcore-call: WalletPool#getWallet","data":["xpub6DBXGC1faXnWSWofjAXjfxihNmYEcJj7KcWytuc7AZK39zS8KdgbosCjnPyHRbwKxcJxnWbzLwZYaPHFE3zoQZmGDbPzwRuoSfFaeSTVwUT_zencash_"],"level":"debug","timestamp":"2020-03-03T15:12:36.063Z"}
+{"message":"libcore-result: WalletPool#getWallet","data":{"value":{}},"level":"debug","timestamp":"2020-03-03T15:12:36.064Z"}
+{"message":"libcore: getOrCreateAccount","data":{"xpub":"xpub6DBXGC1faXnWSWofjAXjfxihNmYEcJj7KcWytuc7AZK39zS8KdgbosCjnPyHRbwKxcJxnWbzLwZYaPHFE3zoQZmGDbPzwRuoSfFaeSTVwUT","index":0},"level":"debug","timestamp":"2020-03-03T15:12:36.065Z"}
+{"message":"libcore-call: Wallet#getAccount","data":[0],"level":"debug","timestamp":"2020-03-03T15:12:36.065Z"}
+{"message":"libcore-result: Wallet#getAccount","data":{"value":{}},"level":"debug","timestamp":"2020-03-03T15:12:36.133Z"}
+{"message":"libcore: sync(1) syncCoreAccount","level":"debug","timestamp":"2020-03-03T15:12:36.133Z"}
+{"message":"libcore-call: Account#synchronize","data":[],"level":"debug","timestamp":"2020-03-03T15:12:36.134Z"}
+```
+
+### A problem during "Add Account"
+
+The _Add Account_ flow involves the [`currencyBridge.scanAccounts`](CurrencyBridge.md) logic which itself is mostly about performing synchronisation. The main problem is that you might not know the XPUB at this point in time and it might breaks before. What we can do easily however is to reuse the APDUs that appears in the log and replay them against the CLI.
+
+On top of the /logsviewer page, there is a handy button "Export APDUs" that we can use to retrieve ALL the apdus. Now, you need to locate the part that concerns the add accounts.
```

Comment:
No explanation of what APDUS are. Maybe we need to explain that they are the messages to communicate to the device ?

Reply:
good point! definitely need a apdu.md, i'll add a link for now